### ISSUE TITLE
Flow Trigger Bundle - can it be made to force a re-load of the item page?

### ISSUE THREAD

**Issue by MarkBurvs** [directus-labs/extensions#72]
### Extension

Something else

### Describe the Improvement

This has potential to be a really useful extension as we have many Flows that we'd love to move into the Item page rather than having to trigger them from the side-bar. 
However, currently, clicking on a Flow panel to trigger a Flow doesn't give you the same experience as triggering one from a button in the side-bar, as it doesn't force a re-load of the page once the Flow is complete. 
Given that most of the Flows we've made cause some sort of modification to the data of the current Item, this means that we  have to manually refresh the page after triggering the Flow in order to see the result. Which isn't really a better experience than the existing experience of triggering Flows from the side-bar which DOES re-load the data and show the result instantly. 

Is this something that could be added?

**Comment by bstrr** [comment]
I second this, (optional) reload + (optional) check for unsaved changes are really needed to make this a great and useful extension.

**Comment by BlackDahlia313** [comment]
I agree. I am running into this very use-case!

The manual flow is designed to bring back scraped data to the current item. I can see this being confusing for a user who thinks they might have to save the item after the flow finishes.

**Comment by dbargaud** [comment]
Same problem here. Could you fix it ?

**Comment by Pedro-Picardi** [comment]
This would be very useful! Upvoting it.

**Comment by rafarc21** [comment]
This would be very helpful for us as well. And it's in fact a duplicate of this issue: https://github.com/directus-labs/extensions/issues/61 which I think could be closed so that activity is focused here ( FYI @bryantgillespie )
